services/nvidia-gpu-operator/1.11.1/defaults/cm.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nvidia-gpu-operator-1.11.1-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |-
```

Comment:
maybe as a follow-up, can we define a security context for the operator? things like run as non-root, read only filesystem, disable privilege escalation?

Reply:
added a note to https://d2iq.atlassian.net/browse/D2IQ-92880